.env.example

```diff
@@ -1,6 +1,9 @@
 DATABASE_URL="postgresql://johndoe:randompassword@localhost:5432/mydb?schema=public" #The database url for the database to be used. Please ensure it is migrated (the tables are created) and optionally seeded (some initial data is inserted). See the README for more information on how to do this.
-PORT= "3001" #The port to run the server on (default is 3001)
 ENCRYPTION_KEY="12345678901234567890123456789012" #This is a secret key used to encrypt the sensitive wallet secrets in the database. Please change this to a secure string of at least 32 characters.
+ADMIN_KEY="abcdef_this_should_be_very_secure" #The key of the admin user, this key will have all permissions, like doing payments, changing configurations and can also be used to create new (more limited) api_keys
```

Comment:
Why did you change this position?

I had it separately, because it is only needed when seeding (meaning not when running)

If you want it there for visibility maybe we can have an `example.seed.env` and an `example.run.env`?